test_data/wordpress_access_logs.txt

```diff
@@ -0,0 +1,19 @@
+{"log":"10.128.0.2 - - [27/Jun/2018:19:29:54 +0000] \"POST /wp-admin/install.php?step=2 HTTP/1.1\" 200 872 \"-\" \"Mozilla/5.0 (Windows; U; Windows NT 10.0) http/2.8.11 Tcl/8.6.7\"\n","strea
```

Comment:
Is this file formatted correctly?  There are newlines mid-line in a lot of places.

Also, wow, that's an ugly log format, :).

Reply:
Yup, this is how Kubernetes logs things when it logs from stdout. I'll change this to "normal" access log fiels